src/sshfs_mount/sshfs_mount.cpp

```diff
@@ -182,10 +181,13 @@ auto make_sftp_server(mp::SSHSession&& session, const std::string& source, const
     set_owner_for(session, leading, missing);
 
     auto output = run_cmd(session, "id -u");
+    mp::utils::trim_newline(output);
```

Comment:
This isn't necessary, `std::stoi` deals with whitespace on its own.

Reply:
Right. In fact, I put there to check for the newline, but I agree it's not needed and the name of the function can be misleading in the sense that it does not specify that it checks for `\n`. I agree to remove it.